base_classes/NXwaveplate.nxdl.xml

```diff
@@ -74,48 +75,53 @@
     </field>
     <field name="wavelengths" type="NX_NUMBER" recommended="true">
         <doc>
-             Discrete wavelengths for which the waveplate is designed. If the
-             waveplate operates over an entire range of wavelengths, enter the minimum
-             and maximum values of the wavelength range (in this case
-             N_wavelengths = 2).
+            Discrete wavelengths for which the waveplate is designed. If the
+            waveplate operates over an entire range of wavelengths, enter the minimum
+            and maximum values of the wavelength range (in this case
+            N_wavelengths = 2).
         </doc>
         <dimensions rank="1">
             <dim index="1" value="N_wavelengths"/>
         </dimensions>
     </field>
+    <group name="retardance_distribution" type="NXdata">
+        <doc>
+            Wavelength resolved retardance of the waveplate.
+        </doc>
+    </group>
     <field name="diameter" type="NX_FLOAT" units="NX_LENGTH">
         <doc>
-             Diameter of the waveplate.
+            Diameter of the waveplate.
```

Comment:
Necessarily circular?

Reply:
I checked if the implementation in the classes and appdefs that was voted upon resolved this comment - it does.